integrated_channels/utils.py

```diff
@@ -161,20 +161,26 @@ def get_image_url(content_metadata_item):
     return image_url
 
 
-def is_already_transmitted(transmission, enterprise_enrollment_id, grade):
+def is_already_transmitted(transmission, enterprise_enrollment_id, grade, subsection_id=None):
     """
     Returns: Boolean indicating if completion date for given enrollment is already sent of not.
 
     Args:
         transmission: TransmissionAudit model to search enrollment in
         enterprise_enrollment_id: enrollment id
         grade: 'Pass' or 'Fail' status
+        subsection_id (Optional): The id of the subsection, needed if there exist multiple audits per course
```

Comment:
multiple audits per course? I get that means the audit table, but my brain was confused for a second there, can we just say for assessments?

Reply:
Sure! I think both are accurate to the situation, multiple assessments and each assessment has an audit. This is `**assessment level reporting**` so I don't think it's a stretch at all to say assessments here.

Reply:
yeah I think basically swapping audit for subsection or assessment might be a little clearer. That or just `multiple assessment audits` but that implies multiple audits per assessment (which might also be possible idk)